phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -1727,4 +1767,26 @@ private int checkGuidePostWidth(String tableName) throws Exception {
         }
     }
 
+    private void createTableWithTableLevelMaxLookbackAge(String fullTableName, String maxLookbackAge) throws Exception {
+        try(Connection conn = DriverManager.getConnection(getUrl())) {
+            String createDdl = "CREATE TABLE " + fullTableName +
+                    " (id char(1) NOT NULL," + " col1 integer NOT NULL," + " col2 bigint NOT NULL," +
+                    " CONSTRAINT NAME_PK PRIMARY KEY (id, col1, col2)) " +
+                    "MAX_LOOKBACK_AGE="+maxLookbackAge;
+            conn.createStatement().execute(createDdl);
+        }
+    }
+
+    private Long queryTableLevelMaxLookbackAge(String schemaName, String dataTableName) throws Exception {
+        try(Connection conn = DriverManager.getConnection(getUrl())) {
```

Comment:
Likewise this would also be simpler using PhoenixRuntime.getTableNoCache() and then looking up from the PTable that's returned.

Reply:
Please use conn.getTableNoCache instead.